src/NewsService.Models.Dto/Responses/NewsResponse.cs

```diff
@@ -11,5 +11,7 @@ public class NewsResponse
         public User Author { get; set; }
         public User Sender { get; set; }
```

Comment:
Should be It edits for author and sender?

Reply:
What do you mean? I did not understand what i can edit in response model